docs/generated/galasactl_runs_get.md

```diff
@@ -1,39 +0,0 @@
-## galasactl runs get
```

Comment:
Accidentally deleted this?

Reply:
Urm. Not consciously. If this is generated, then it must be re-generated surely ?